.github/workflows/test.yml

```diff
@@ -37,12 +37,12 @@ jobs:
           - '22.0'
           - '20'
           - '20.0'
-          - '18.18.2' # Skip >=18.19.0 until IITM issues are resolved.
+          - '18'
```

Comment:
reviewer note: The IITM fix was a while back, so we are good testing with latest Node.js v18 now.